Cargo.toml

```diff
@@ -29,6 +29,7 @@ cassowary = "0.3"
 compact_str = "0.8.0"
 crossterm = { version = "0.28", optional = true }
 document-features = { version = "0.2.7", optional = true }
+indoc = "2"
```

Comment:
I don't recall exactly why we removed indoc from the main deps, but it's possible we'd need to avoid it here (I think it was possibly due to someone running on android). Check before any potential merge.

Reply:
for context, it was removed in this PR: https://github.com/ratatui-org/ratatui/pull/1007